front-end/src/shared/cancel-icon.svg

```diff
@@ -0,0 +1,40 @@
+<!-- Source: Alfredo Hernandez, Flaticon  https://www.flaticon.com/free-icon/close_463612?term=cancel&page=1&position=22 -->
```

Comment:
Can we turn this svg into its own React component? I think the logo is an example that you can use.

Reply:
Changed into a React component and cleaned up the svg, seemed like those excess g nodes didnt do anything.